File: Examples/CSharp/WorkingWithProjects/ReadProjectFiles.cs
namespace Aspose.Tasks.Examples.CSharp.WorkingWithProjects
{
    public class ReadProjectFiles
    {
        public static void Run()
        {
            // ExStart:ReadProjectFiles
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_WorkingWithProjects();        

            Project project = new Project(dataDir + "ReadProjectFiles.mpp");
            // ExEnd:ReadProjectFiles
        }
    }
}

File: Examples/VisualBasic/WorkingWithProjects/CreatingReadingAndSaving/ReadProjectFiles.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Tasks
Imports System

Namespace WorkingWithProjects.CreatingReadingAndSaving
    Public Class ReadProjectFiles
        Public Shared Sub Run()
            ' ExStart: ReadProjectFiles
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Read existing project template file
            Dim project As New Project(dataDir & "ReadProjectFiles.mpp")
            ' ExEnd: ReadProjectFiles
        End Sub
    End Class
End Namespace